test/test_screen.py

```diff
@@ -15,24 +15,26 @@ def setUpClass(self):
         set_root(__file__)
 
     def setUp(self):
-        surf.fill((0, 0, 0))
         self.screen = Screen(surf)
+        self.screen.clear()
 
-    def assertImagesEqual(self, a, b):
-        adata, bdata = (pygame.image.tostring(i, 'RGB') for i in (a, b))
+    def assertAlmostImagesEqual(self, a, b):
+        zdata = zip(pygame.image.tostring(a, 'RGB'), pygame.image.tostring(b, 'RGB'))
 
-        if adata != bdata:
-            raise AssertionError("Images differ")
+        for z in zdata:
+            # TODO: Comment on when can those differences appear and why do we allow them
```

Comment:
@lordmauve You've suggested that this might be the cause for the tests to fail and it really is. Any hints on why is that happening? I'd like to comment on that in the code.

Reply:
This could be due simply to the implementation of alpha blending changing slightly.

The alpha-blending operation is 

```
dest = source_alpha * source + (1 .0 - source_alpha) * dest
```

for source_alpha in `[0.0, 1.0]`. Depending on how you do you implement that operation - eg. premultiplying alpha, using floats, rounding the result - you could get small differences in the low bits.

Reply:
@lordmauve Thanks! I'll fix that TODO comment in the code.

Reply:
Fixed in https://github.com/lordmauve/pgzero/pull/102/files#diff-52d7f9ea4367153211e452af4567526aR22